examples/flutter_gallery/lib/demo/cupertino/cupertino_dialog_demo.dart

```diff
@@ -73,10 +73,12 @@ class _CupertinoDialogDemoState extends State<CupertinoDialogDemo> {
               showDemoDialog<String>(
                 context: context,
                 child: new CupertinoAlertDialog(
-                  title: const Text('Allow "Maps" to access your location while you use the app?'),
+                  title: const Text('Allow "Maps" to access your location while'
+                      ' you use the app?'),
                   content: const Text(
-                    'Your current location will be displayed on the map and used for directions, '
-                    'nearby search results, and estimated travel times.'
+                    'Your current location will be displayed on the map and '
+                        'used for directions, nearby search results, and '
+                        'estimated travel times.'
```

Comment:
ditto

Reply:
also, subsequent lines should be vertically aligned with the first

Reply:
I'll update accordingly.